.github/workflows/reusable-build-test-release.yml

```
@@ -1864,6 +2051,23 @@ jobs:
           path: |
             ${{ needs.setup.outputs.directory-path }}/diag*
 
+  Modinput-tests-report:
+    needs: run-modinput-tests
+    runs-on: ubuntu-latest
+    if: ${{ !cancelled() && needs.run-modinput-tests.result != 'skipped' }}
+    steps:
+      - name: Download all summaries
+        uses: actions/download-artifact@v4
+        with:
+          pattern: summary-modinput*
+      - name: Combine summaries into a table
+        run: |
+          echo "| Job | Total Tests | Passed | Failed  | Errors| Skipped | Report Link" >> "$GITHUB_STEP_SUMMARY"
+          echo "| ---------- | ----------- | ------ | ------ | ------ | ------- | ------ |" >> "$GITHUB_STEP_SUMMARY"
+          for file in summary-modinput-*/job_summary.txt; do
+            cat "$file" >> "$GITHUB_STEP_SUMMARY"
+          done
```

Review artifacts that are uploaded and no longer needed after workflow execution - consider deleting them

```suggestion
      - name: Combine summaries into a table
        run: |
          echo "| Job | Total Tests | Passed | Failed  | Errors| Skipped | Report Link" >> "$GITHUB_STEP_SUMMARY"
          echo "| ---------- | ----------- | ------ | ------ | ------ | ------- | ------ |" >> "$GITHUB_STEP_SUMMARY"
          for file in summary-modinput-*/job_summary.txt; do
            cat "$file" >> "$GITHUB_STEP_SUMMARY"
          done
      - uses: geekyeggo/delete-artifact@v5
        with:
          name: summary-modinput-${{ matrix.splunk.version }}-${{ secrets.OTHER_TA_REQUIRED_CONFIGS }}-${{ matrix.vendor-version.image }}-${{ matrix.marker }}-artifact
```

---

Done